server/testdriver/core/src/main/java/org/infinispan/server/test/core/persistence/ContainerDatabase.java

```diff
@@ -63,6 +64,7 @@ public void stop() {
       log.infof("Stopping database %s", getType());
       container.stop();
       log.infof("Stopped database %s", getType());
+      ContainerInfinispanServerDriver.removeDockerImage(container.getDockerImageName());
```

Comment:
Do we really need to build an image for the database containers?

All we do with the image is

.expose(port)
.env(env)

I don't think expose is needed -- we can just use the database's default port
And we can use `withEnv()` on the container instead of `env()` on the image.